index.bs

```diff
@@ -8546,6 +8564,23 @@ entails.
 
 <h4 id="LegacyWindowAlias" extended-attribute lt="LegacyWindowAlias">[LegacyWindowAlias]</h4>
 
+<div class="advisement">
+
+    [{{LegacyWindowAlias}}] [=extended attributes=] are an undesirable feature.
+    They exist only so that legacy Web platform features can be specified.
+    They should not be used in specifications
+    unless required to specify the behavior of legacy APIs.
+
+    <small class="non-normative">
+        The [{{LegacyWindowAlias}}] [=extended attribute=] appears on the following [=interfaces=]:
```

Comment:
I'm happy to meet you in the middle, however, and have an exhaustive list which isn't presented as such.